### ISSUE TITLE
MIC seg got much lower miou than you have described in the paper

### ISSUE THREAD

**Issue by DataSttructure** [lhoyer/MIC#17]
hi lukas
I trained the MIC segmentation according to your code. But something seems to have gone wrong, the miou is much lower than what you describe in your paper. 
Here is the log output. Can you help me analyze the reason? Thank you! 
[20230306_143102.log](https://github.com/lhoyer/MIC/files/10926905/20230306_143102.log)

**Comment by lhoyer** [comment]
Hi @DataSttructure,

Thank you for your interest in our work. It seems that your system uses Cuda 11.3 while pytorch and mmcv were compiled with Cuda 11.1. Please, refer to issue #8. The simplified environment setup https://github.com/lhoyer/MIC/issues/8#issuecomment-1383777610 and possibly changing the random seed as discussed in #9 should solve this issue.

**Comment by DataSttructure** [comment]
Sorry to bother you again. I tried setup a environment as you mentioned in [#8 (comment)](https://github.com/lhoyer/MIC/issues/8#issuecomment-1383777610) in a Docker container. 
But there is still the problem of very low train class IoU. 
[20230314_062715.log](https://github.com/lhoyer/MIC/files/10997183/20230314_062715.log)

**Comment by DataSttructure** [comment]
In addition, I changed the cuda version to 11.0 in the Docker container.

**Comment by lhoyer** [comment]
Can you please try to change the random seed as discussed in https://github.com/lhoyer/MIC/issues/9?

**Comment by jetitime** [comment]
Dear @lhoyer ,

Sorry to disturb you. I tried once using the command : python run_experiments.py --config configs/mic/gtaHR2csHR_mic_hrda.py , and get a result around 20 mIoU even after 40000 iterations, which is much lower than the results in paper. My latest log is  [20230520_101036.log(https://github.com/lhoyer/MIC/files/11521633/20230520_101036.log)
  Looking forward to your advise.

Best,
Jetitime
